### PR TITLE
emojis but the branch isn't deleted

### PR DESCRIPTION
**Pull request by LetterN**
:snya:

### CODE REVIEW

File: code/controllers/settings/game_options.dm [hunk 1]
```diff
@@ -30,6 +30,8 @@
 	var/remove_gun_restrictions = 0
 	var/allow_synthetic_gun_use = 0
 
+	var/emojis = 0
```
Comment (ninjanomnom) [inline code/controllers/settings/game_options.dm]: FALSE

File: code/controllers/settings/game_options.dm [hunk 2]
```diff
@@ -75,5 +77,8 @@
 			config.use_loyalty_implants = 1
 		if("remove_gun_restrictions")
 			config.remove_gun_restrictions = 1
+
+		if("emojis")
+			config.emojis = 1
```
Comment (ninjanomnom) [inline code/controllers/settings/game_options.dm]: TRUE

File: config/game_options.txt [hunk 3]
```diff
@@ -61,6 +61,9 @@ ANIMAL_DELAY 0
 ### Miscellaneous ###
 
 ## Config options which, of course, don't fit into previous categories.
+# Remove the # in front of this config option to have loyalty implants spawn by default on your server.
+#USE_LOYALTY_IMPLANTS
 
-## Remove the # in front of this config option to have loyalty implants spawn by default on your server.
```
Comment (ninjanomnom) [inline config/game_options.txt]: You seem to have removed a `#` by mistake here for no reason

File: code/modules/admin/verbs/adminpm.dm [hunk 4]
```diff
@@ -36,41 +36,46 @@
...
+	msg = emoji_parse(msg)
 	var/recieve_color = "purple"
-	var/send_pm_type = " "
+	var/send_pm_type = ""
 	var/recieve_pm_type = "Player"
 
 
-	if(holder)
-		//PMs sent from admins and mods display their rank
+	if(C.holder) //PMs sent from admins and mods display their rank
```
Comment (Rohesie) [inline code/modules/admin/verbs/adminpm.dm]: This is never triggering. Staff can't reply to AHelps:
https://i.gyazo.com/9b4e202090d044f196b4cd2c83973adf.png

**Comment by Rohesie** [comment]
This has been testmerged and didn't work, besides breaking the aforementioned thing. Please fix.

**Comment by Rohesie** [comment]
Feel free to ask this to be re-opened when you resume working on it.

**Pull request closed without merging**